src/client/js/main.js

```diff
@@ -1,5 +1,23 @@
-// 아래는 메인 페이지 관련 js만드셨 을때 예제입니다. 주석을 지우고 하시면 됩니다.
-
 import "../scss/styles.scss";
+import Category from "./category.js";
+import FloatingButton from "./floatingButton.js";
+
+const categoryComponent = new Category(document.querySelector(".categoryList"));
+
+if(document.querySelector(".main-wrap section").scrollHeight > window.innerHeight - 359) {
+  const floatingButton = new FloatingButton().getButton();
+  floatingButton.className = "main_floatingButton";
+  document.querySelector("body").appendChild(floatingButton);
 
+  floatingButton.addEventListener('click', (e) => {
+    if(e.currentTarget === floatingButton) {
+      document.scrollingElement.scrollTo(0, 0);
+    }
+  })
 
+  // Refactor: 성능 개선
+  document.addEventListener('scroll', e => {
+    if(document.scrollingElement.scrollTop === 0) floatingButton.style.visibility = 'hidden';
```

Comment:
이 부분도 개인의 코드 스타일 나름이지만 일반적으로는 가독성을 위해 한줄로 끝나는 조건문일지라도 아래와 같이 적어주곤 한답니다.  아래 else도 동일
물론 취향이긴 하니, 참고만 해주세요
```suggestion
    if (document.scrollingElement.scrollTop === 0) {
      floatingButton.style.visibility = 'hidden';
      }
```

Reply:
해당 글은 컨벤션에서 읽어본 적 있습니다.
1줄짜리여도 괄호 처리해주는 것을 더 추천했었죠..!!!!
적용하겠습니당